### PR TITLE
Feature test

### PR DESCRIPTION
**Pull request by neobay991**
- Update to Podfile to we have one for Feature and Unit Tests
- Update to build settings
- Feature tests for Sign up, Login, user posting a message

### CODE REVIEW

File: messenger-appUITests/messenger_appUITests.swift [hunk 1]
```diff
@@ -1,36 +1,207 @@
 //
-//  messenger_appUITests.swift
+//  messenger_appUITests1.swift
```
Comment (heuperman) [inline messenger-appUITests/messenger_appUITests.swift]: That 1 at the end is a leftover?

File: messenger-appUITests/messenger_appUITests.swift [hunk 2]
```diff
@@ -1,36 +1,207 @@
...
+        app.buttons["Sign Up"].tap()
         
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+        XCTAssert(app.buttons["Sign Up"].exists)
+        XCTAssert(app.textFields["Email"].exists)
+        XCTAssert(app.textFields["Password"].exists)
+        XCTAssert(app.buttons["OK"].exists)
+    }
+    
+    // It checks the login up form exists
```
Comment (heuperman) [inline messenger-appUITests/messenger_appUITests.swift]: login up form => login form

File: messenger-appUITests/messenger_appUITests.swift [hunk 3]
```diff
@@ -1,36 +1,207 @@
...
+        let emailTextField = app.textFields["Email"]
+        emailTextField.tap()
+        
+        let passwordTextField = app.textFields["Password"]
+        passwordTextField.tap()
+        passwordTextField.typeText("password")
+        
+        app.buttons["OK"].tap()
+        
+        XCTAssert(app.buttons["OK"].exists)
```
Comment (heuperman) [inline messenger-appUITests/messenger_appUITests.swift]: For now this is fine but once we figure out how we should mock the database so we don't fill it up with test accounts (we're limited to 100 users)

File: messenger-appUITests/messenger_appUITests.swift (hunk 1 above)
Comment (neobay991) [reply messenger-appUITests/messenger_appUITests.swift]: Good spot, missed updating the comment 👍

File: messenger-appUITests/messenger_appUITests.swift (hunk 2 above)
Comment (neobay991) [reply messenger-appUITests/messenger_appUITests.swift]: 👍

File: messenger-appUITests/messenger_appUITests.swift (hunk 3 above)
Comment (neobay991) [reply messenger-appUITests/messenger_appUITests.swift]: yep, I did some research but could not find an obvious answer 🙏 . Ideally for Feature tests we mock or destroy the database and records and recreate for each test like Ruby/ Rails

**Pull request merged**